vivisect/base.py

```diff
@@ -696,7 +696,7 @@ def trackDynBranches(cfctx, op, vw, bflags, branches):
     if len(vw.getXrefsFrom(op.va)):
         return
 
-    vw.vprint("0x%x: Dynamic Branch found at (%s)" % (op.va, op))
+    logger.info("0x%x: Dynamic Branch found at (%s)" % (op.va, op))
```

Comment:
We really need to have a conversation about the vw.vprint vs logger split we've got going on (but not in this PR)

Reply:
we do.  i'd love to get some strategy we agree on.  this one has always seemed like noise to me (not to me, but to others) so logging seemed better (which we didn't really have back when we wrote this code originally).

Reply:
did you prefer this as a vprint?  i can put it back.